src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -255,9 +255,9 @@ public bool TryGetWellKnownAttribute(WellKnownAttributeFlags flag, out bool valu
 
                 // CONSIDER: Can we make parameter type computation lazy?
                 var typeSymbol = DynamicTypeDecoder.TransformType(type.TypeSymbol, countOfCustomModifiers, handle, moduleSymbol, refKind);
-                typeSymbol = TupleTypeDecoder.DecodeTupleTypesIfApplicable(typeSymbol, handle, moduleSymbol);
                 type = type.Update(typeSymbol, type.CustomModifiers);
-                type = NullableTypeDecoder.TransformOrEraseNullability(type, _handle, moduleSymbol, extraAnnotations);
+                type = TupleTypeDecoder.DecodeTupleTypesIfApplicable(type, handle, moduleSymbol);
```

Comment:
Nit: for consistency with other locations, I'd move the tuple handling after the handling for nullable.